src/main/resources/portal.properties.EXAMPLE

```diff
@@ -202,3 +202,13 @@ ucsc.build=hg19
 
 # default view in oncoprint (sample, patient (default))
 oncoprint.defaultview=patient
+
+# labels to be displayed in oncoprint "Mutation color" menu for custom annotation of driver and passenger mutations in the oncoprint.
+# Set these properties to enable the respective menu options in oncoprint:
+# oncoprint.custom_driver_annotation.binary.menu_label=Custom binary annotation
+# oncoprint.custom_driver_annotation.class.menu_label=Custom tiers
+
+# set this to false to disable the automatic selection of the custom driver and passenger mutations annotation in the oncoprint 
+# (instead of OncoKB and Hotspots). By default this is true. Setting this to false will select OncoKB and Hotspots as the default 
+# annotation source instead.
+# oncoprint.custom_driver_annotation.default=false
```

Comment:
I think it makes sense to have the icons, they just give information about the mutation stored in the database. If there is information about OncoKB, the icon is also shown there (even if you have the OncoKB annotation disabled). I think the tooltip and the menu should be seen as related, but separated parts.